values/oauth2-proxy/oauth2-proxy.gotmpl

```diff
@@ -110,7 +110,20 @@ redis:
       sidecar.istio.io/inject: "false"
     priorityClassName: otomi-critical
     resources: {{- $r.resources.master | toYaml | nindent 6 }}
-
+    podLabels:
+      otomi.io/affinity: "anti"
+    affinity:
```

Comment:
This replaces the (positive) default pod affinity matching labels:
```
matchLabels:                                
  app.kubernetes.io/instance: oauth2-proxy  
  app.kubernetes.io/name: redis             
  app.kubernetes.io/component: master       
```
I currently do not even see what this would be even good for (more than one master for the same Redis instance on the same node?). Just wanted to double-check if you have an idea about this, and if this could become an issue.